### PR TITLE
Update htmlunit to current version for examples

### PR DESCRIPTION
**Pull request by erdlet**
see: #115
Signed-off-by: Erdle, Tobias <tobias.erdle@innoq.com>

### CODE REVIEW

File: examples/pom.xml [hunk 1]
```diff
@@ -49,7 +49,7 @@
         <dependency>
             <groupId>net.sourceforge.htmlunit</groupId>
             <artifactId>htmlunit</artifactId>
-            <version>2.15</version>
+            <version>2.36.0</version>
```
Comment (chkal) [inline examples/pom.xml]: Do we already have a CQ for this new version?

Comment (erdlet) [reply examples/pom.xml]: Yes, a few of them are already processed and certified. For htmlunit it's this one:
https://dev.eclipse.org/ipzilla/show_bug.cgi?id=21353

Comment (chkal) [reply examples/pom.xml]: Ok, great! Thanks for sharing the link.

**Comment by erdlet** [comment]
Short hint: I track the CQs in the issue. All approved ones are marked by a
green check :)

Christian Kaltepoth <notifications@github.com> schrieb am Do., 9. Jan.
2020, 21:15:

**Comment by chkal** [comment]
> Short hint: I track the CQs in the issue. All approved ones are marked by a green check :)

Awesome! I forgot to check this issue for the latest status. :+1:

**Pull request merged**